src/ripple/ripple.js

```diff
@@ -118,7 +119,12 @@ MaterialRipple.prototype.upHandler_ = function(event) {
 
   // Don't fire for the artificial "mouseup" generated by a double-click.
   if (event && event.detail !== 2) {
-    this.rippleElement_.classList.remove(this.CssClasses_.IS_VISIBLE);
+    // Allow a repaint to occur before removing this class, so the animation
+    // shows for tap events, which seem to trigger a mouseup too soon after
+    // mousedown.
+    window.setTimeout(function() {
```

Comment:
@samccone I’d agree, but we are not actually waiting for the next frame, we just want to push some code back on the queue so we can yield to the browser.

@cainmi Would you mind changing to `setTimeout(...,0)`, than this is good to go :) Thanks for your patience, this blew up a little :boom: